core/src/main/java/org/ehcache/core/spi/cache/AbstractValueHolder.java

```diff
@@ -181,6 +183,6 @@ public long getId() {
 
   @Override
   public String toString() {
-    return "" + value();
+    return format("%s", value());
```

Comment:
Will need something else to understand what you would expect? Last option from me is just to delete